client/actions/eventActions.js

```diff
@@ -73,7 +73,10 @@ export function getEventSelected(id, tag) {
 }
 
 export function modifyCenterEvent(data) {
-  const { id, centerId } = data;
+  const { 
+    id,
+    centerId
+  } = data;
   return (dispatch) => {
     dispatch({ type: 'MODIFY_CENTER_EVENT' });
```

Comment:
You might want to create a file that holds your `actionTypes`assigned to constants, and then import them whenever you need to dispatch actions. It is safer that way to avoid typos when you have to type the action types as strings everytime you need to dispatch an action.